plugins/fileservices/fileservices.cpp

```diff
@@ -1141,6 +1179,9 @@ FILESERVICES_API char * FILESERVICES_CALL implementSprayJson(ICodeContext *ctx,
     if (noSplit)
         req->setNosplit(true);
 
+    if (keepSourceEncoding)
+        req->setKeepSourceEncoding(true);
```

Comment:
as per last comment (and code in implementSprayVariable) think you need else statement here.

Reply:
Else added